tests/configuration/test_secret_file.py

```diff
@@ -0,0 +1,65 @@
+"""Tests for SecretFileValue"""
+
+import os
+
+import pytest
+
+from lasuite.configuration.values import SecretFileValue
+
+FILE_SECRET_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), "test_secret")
+
+
+@pytest.fixture
```

Comment:
```suggestion
@pytest.fixture(autouse=True)
```

And you don't need to use it anymore in your tests